packages/run-protocol/src/vaultFactory/vaultFactory.js

```diff
@@ -39,12 +39,26 @@ import { makeVaultParamManager, makeElectorateParamManager } from './params.js';
 const { details: X } = assert;
 
 /**
- * @param {ZCF<Record<string, any> &
+ * @typedef {{
```

Comment:
It would be nicer (lower maintenance, etc.) if this inferred the type.

Reply:
okey doke 8fdfcea4bda55cc7c3448c1c6c1e67fd5d5ec683